packages/flutter_tools/test/general.shard/build_system/source_test.dart

```diff
@@ -13,7 +13,7 @@ import 'package:flutter_tools/src/build_system/exceptions.dart';
 import 'package:flutter_tools/src/globals.dart' as globals;
 
 import '../../src/common.dart';
-import '../../src/context.dart';
+import '../../src/fake_process_manager.dart';
```

Comment:
Not related, but noticed this file was just using the exported `fake_process_manager`, which is null safe.